### PR TITLE
tmsis -r now shows times as seconds since epoch

### PR DESCRIPTION
**Pull request by kkroo**
In requesting the raw TMSI table output, give times as seconds since the epoch rather than `prettyAge`d times

### CODE REVIEW

File: Control/TMSITable.cpp [hunk 1]
```diff
@@ -819,13 +819,21 @@ vector< vector<string> > TMSITable::tmsiTabView(int verbosity, bool rawFlag, uns
 				}
 			} else if (header == "CREATED" || header == "ACCESSED") {
 				// Print seconds as a time value.
-				row.push_back(prettyAge(now - query.getResultInt(col)));
+				if (rawFlag) {
+					row.push_back(format("%d", query.getResultInt(col)));
```
Comment (shaddi) [inline Control/TMSITable.cpp]: Shouldn't this also be `now - query.getResultInt(col)`? Seems like raw should not change the semantics of the field.

Comment (kkroo) [inline Control/TMSITable.cpp]: It depends on how you interpret what the raw flag should do. The description says it's spitting out DB values, but I think giving the deltas is acceptable and would eliminate possible issues with unsynchronized clocks.

Comment (shaddi) [inline Control/TMSITable.cpp]: Ah, interesting point. Your call what the semantics should be, but
regardless let's update the documentation to state that explicitly.

**Comment by shaddi** [comment]
lgtm

**Pull request merged**